src/main/resources/assets/create/lang/es_es.json

```diff
@@ -30,13 +30,13 @@
 	"block.create.black_sail": "Vela negra",
 	"block.create.black_seat": "Asiento negro",
 	"block.create.black_valve_handle": "Asa de válvula negra",
-	"block.create.blaze_burner": "Quemador de Blaze",
+	"block.create.blaze_burner": "Blaze Burner",
 	"block.create.blue_sail": "Vela azul",
 	"block.create.blue_seat": "Asiento azul",
 	"block.create.blue_valve_handle": "Asa de válvula azul",
 	"block.create.brass_belt_funnel": "Cinturón de embudo de bronce",
 	"block.create.brass_block": "Bloque de bronce",
-	"block.create.brass_casing": "Revestidor de bronce",
+	"block.create.brass_casing": "Recubridor de bronce",
```

Comment:
I put revestidor because it derives from revestimiento and it is easier to remember.

Reply:
i didn't make the change, you did on your last update, sorry about this